src/components/ProgramRecordsList.jsx

```diff
@@ -21,6 +23,14 @@ const ProgramRecordsList = () => {
       if (_.isEmpty(data)) {
         setHasNoData(true);
       } else {
+        (data.enrolled_programs).map((program) => { /* eslint-disable no-param-reassign */
+          if (program.completed) {
+            program.status = 'Completed';
+          } else {
+            program.status = 'Partially Completed';
```

Comment:
Are these values displayed anywhere? Do they to support translation?

Reply:
They are a field added to our JSON object, which is used for building our DataTable (line 124), but they are not displayed as plain text anywhere. I don't believe they need to support translation, unless we somehow need to support translation for our entire table.